modules/simple_forms_api/app/form_mappings/vba_21_0972.json.erb

```diff
@@ -0,0 +1,64 @@
+{
+  "F[0].#subform[1].Email_Address[0]": "<%= data.dig('claimant_email') %>",
```

Comment:
nit: anywhere you are digging with one layer you can just do `data['key']`. It is nil safe.
It won't throw an error in here because it's masked by the json, but in other ruby files it's a linting thing because you can simplify it to not use the dig.